### PR TITLE
Assign lieutenancy

### PR DESCRIPTION
**Pull request by dreamfall**
https://app.asana.com/0/1200175839265057/1200560530477251

### CODE REVIEW

File: app/controllers/admin/lieutenant_assignment_collections_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Admin::LieutenantAssignmentCollectionsController < Admin::BaseController
+  def create
+    puts create_params
```
Comment (TheDancingClown) [inline app/controllers/admin/lieutenant_assignment_collections_controller.rb]: Remove puts

File: app/policies/form_answer_policy.rb [hunk 2]
```diff
@@ -126,4 +130,8 @@ def can_download_original_pdf_of_application_before_deadline?
   def can_add_collaborators_to_application?
     admin?
   end
+
+  def lieuenancy_assigned?
```
Comment (TheDancingClown) [inline app/policies/form_answer_policy.rb:134]: :zany_face:

**Review by TheDancingClown — CHANGES_REQUESTED** [review]
Just the puts statement to take out then looks good

File: app/policies/form_answer_policy.rb (hunk 2 above)
Comment (dreamfall) [reply app/policies/form_answer_policy.rb]: 🤪

**Review by TheDancingClown — APPROVED** [review]

**Pull request merged**